editor/editor_help.cpp

```diff
@@ -2256,7 +2256,7 @@ void EditorHelp::generate_doc() {
 		doc = memnew(DocTools);
 	}
 
-	if (first_attempt && FileAccess::exists(get_cache_full_path())) {
+	if (first_attempt && FileAccess::exists(get_cache_full_path()) && p_use_cache) {
```

Comment:
Nitpick, but you could check `p_use_cache` earlier so we don't need to access the filesystem if not needed.